Components/AccordionVertical/script.js

```diff
@@ -0,0 +1,130 @@
+import $ from 'jquery'
+
+class AccordionVertical extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    self.setOptions()
+    self.resolveElements()
+    return self
+  }
+
+  setOptions () {
+    this.aria = {
+      hidden: 'aria-hidden',
+      controls: 'aria-controls',
+      expanded: 'aria-expanded'
+    }
+    this.class = {
+      expanded: 'accordionVertical-item-isExpanded',
+      hidden: 'accordionVertical-item-isHidden',
+      contentIsSizing: 'accordionVertical-content-isSizing'
+    }
+    this.isMultiselectable = this.$.attr('aria-multiselectable')
+  }
+
+  resolveElements () {
+    this.$item = $('.accordionVertical-item', this)
+    this.$button = $('.accordionVertical-btn[aria-controls]', this)
+    this.$content = $('.accordionVertical-content', this)
+  }
+
+  connectedCallback () {
+    this.$.on('click', this.$button.selector, this.toggleButton)
+    $(window).on('load resize', this.updateExpandedItemHeight)
+  }
+
+  /**
+   * Toggle item state for clicked button.
+   */
+  toggleButton = (e, expanded) => {
+    e.preventDefault()
+    const $button = $(e.currentTarget)
+    this.toggleItem($button, expanded)
+  }
+
+  /**
+   * Toggle an item's "expanded" state, optionally providing a target
+   * state.
+   *
+   * @param {HTMLButtonElement} button
+   * @param {boolean?} expanded If no state is provided, the current
+   * state will be toggled (from false to true, and vice-versa).
+   * @return {boolean} the resulting state.
+   */
+  toggleItem ($button, expanded) {
+    const id = $button.attr(this.aria.controls)
+    const $item = $button.closest(this.$item.selector)
+    const $content = $(`#${id}`, this)
+
+    if (!$content.length) {
+      throw new Error(
```

Comment:
Are you sure you want to throw an error here and not `console.error` and then return? Though I'm actually not sure if this error only stops execution in this function or the whole file. Maybe you've found this to be better while testing?